contracts/modules/legacy/LegacyPooledStaking.sol

```diff
@@ -1418,16 +1278,20 @@ contract LegacyPooledStaking is IPooledStaking, MasterAwareV2 {
     params = new ProductInitializationParams[](migratableCount);
     uint migrateAtIndex = 0;
 
+
     for (uint i = 0; i < contractCount; i++) {
       if (stakes[i] == 0) {
         continue;
       }
       uint96 price = getV1PriceForProduct(products[i]);
+      if (price == type(uint96).max) {
+        continue;
```

Comment:
skipping product that does not have a price; read above in the getV1PriceForProduct function